### DISCUSSION TITLE
hide value except last 4

### DISCUSSION THREAD

**Discussion by vinodhum** [react-hook-form/react-hook-form#2113] [Q&A]
Hi,
I am using controller with custom input, everything is working as expected. 
How can I hide the value of input except last 4 char, on submit I need actually value
value is hidden only when its focus, how can I achieve this with react from hook using controller ?

**Comment by bluebill1049** [comment]
can you provide a codesandbox?

**Comment by vinodhum** [comment]
https://codesandbox.io/s/quiet-sound-ewohk?file=/src/App.js
for some reason I could not get ionic input work with controller on this example

**Comment by vinodhum** [comment]
any update on this how to achieve it ?

**Reply by bluebill1049** [reply]
sorry, completely missed your last reply.

**Reply by bluebill1049** [reply]
wait your codesandbox is empty...

**Reply by bluebill1049** [reply]
is your question related to a mask input?
